app/templates/eventView.html

```diff
@@ -3,6 +3,7 @@
 {% set isPastEnd = event.isPastEnd %} 
 {% set tabName = 'view' %}
 {% extends "eventNav.html"%}
+{% set eventPast = event.isPastStart if not isNewEvent else False %}
```

Comment:
Rather than add `eventPast` here, when it is used once later, let's just make that conditional use `event.isPastStart`

Reply:
This is a good case for a comment explaining why we need to set eventPast here.